package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "bedrock-zcap-storage-http",
+  "version": "0.0.1-0",
+  "description": "Bedrock Authorization Capability Storage HTTP API",
+  "main": "./lib",
+  "scripts": {},
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/digitalbazaar/bedrock-zcap-storage-http"
+  },
+  "keywords": [
+    "bedrock",
+    "zcap",
+    "ocap",
+    "ocap-ld",
+    "authorization",
+    "capability",
+    "authorization capability"
+  ],
+  "author": {
+    "name": "Digital Bazaar, Inc.",
+    "email": "support@digitalbazaar.com",
+    "url": "http://digitalbazaar.com"
+  },
+  "bugs": {
+    "url": "https://github.com/digitalbazaar/bedrock-zcap-storage-http/issues"
+  },
+  "homepage": "https://github.com/digitalbazaar/bedrock-zcap-storage-http",
+  "dependencies": {
+    "assert-plus": "^1.0.0"
+  },
+  "peerDependencies": {
+    "bedrock": "^3.0.0",
+    "bedrock-account": "^2.1.0",
+    "bedrock-express": "^3.0.0",
+    "bedrock-passport": "^5.0.0",
+    "bedrock-permission": "^2.5.0",
+    "bedrock-mongodb": "^6.0.0  ",
```

Comment:
- space at the end.
- do we need to support ranges for peer deps here or just assume we're only using newer peer deps?